server/app/models/stack.rb

```diff
@@ -16,6 +17,7 @@ class Stack
   index({ grid_id: 1 })
   index({ name: 1 })
   index({ parent_name: 1 })
+  index({ labels: 1 })
```

Comment:
This requires a migration (see `db/migrations/`).

Reply:
basically just `def up; Stacky.create_indexes; end`

Reply:
thanks for the hint - will update migrations to reflect the change.

Reply:
✅ RESOLVED